packages/emotion/src/__tests__/useTheme.test.tsx

```diff
@@ -100,78 +100,11 @@ describe('useTheme hook', () => {
 
       expect(callback).to.have.been.calledWith(canvasHighContrast)
     })
-    it('should override the default theme when no other theme is available', async () => {
-      const callback = spy()
-      await mount(
-        <InstUISettingsProvider theme={{ colors: { brand: 'red' } }}>
-          <ExampleComponent callback={callback}></ExampleComponent>
-        </InstUISettingsProvider>
-      )
-
-      expect(callback).to.have.been.calledWithMatch({
-        ...canvas,
-        colors: { brand: 'red' }
-      })
-    })
-    it('should override the global theme if that is the only theme available', async () => {
-      const callback = spy()
-      const theme = ThemeRegistry.registerTheme(instructure)
-      theme.use()
-
-      await mount(
-        <InstUISettingsProvider theme={{ colors: { brand: 'red' } }}>
-          <ExampleComponent callback={callback}></ExampleComponent>
-        </InstUISettingsProvider>
-      )
-
-      expect(callback).to.have.been.calledWithMatch({
-        ...theme,
-        colors: { brand: 'red' }
-      })
-    })
-    it('should override the global theme with the "themeOverrides" option if that is the only theme available', async () => {
-      const callback = spy()
-      const theme = ThemeRegistry.registerTheme(instructure)
-      theme.use()
-
-      await mount(
-        <InstUISettingsProvider
-          theme={{ themeOverrides: { colors: { brand: 'red' } } }}
-        >
-          <ExampleComponent callback={callback}></ExampleComponent>
-        </InstUISettingsProvider>
-      )
-
-      expect(callback).to.have.been.calledWithMatch({
-        ...theme,
-        colors: { brand: 'red' }
-      })
-    })
-    it('should override the theme from context when provided', async () => {
-      const callback = spy()
-      const theme = ThemeRegistry.registerTheme(instructure)
-      theme.use()
-
-      await mount(
-        <InstUISettingsProvider theme={canvasHighContrast}>
-          <InstUISettingsProvider
-            theme={{ themeOverrides: { colors: { brand: 'red' } } }}
-          >
-            <ExampleComponent callback={callback}></ExampleComponent>
-          </InstUISettingsProvider>
-        </InstUISettingsProvider>
-      )
-
-      expect(callback).to.have.been.calledWithMatch({
-        ...canvasHighContrast,
-        colors: { brand: 'red' }
-      })
-    })
```

Comment:
please do not delete these, here we are not testing for something that can be done by Chromatic, but the business logic of the theming engine